rust/src/pgsql/pgsql.rs

```diff
@@ -296,6 +322,7 @@ impl PgsqlState {
             PgsqlFEMessage::UnknownMessageType(_) => {
                 SCLogDebug!("Match: Unknown request message type");
                 // Not changing state when we don't know the message
+                // TODO I'm not sure None is what we should return here
```

Comment:
A TODO to analyze this better. Probably not an issue now, as we're not logging UnknownMessages at this point, but something that shouldn't go totally off the radar. Thus, the TODO comment

Reply:
I think you should log the type/identifier of unknown messages ;-)